python/lsst/daf/base/exampleLib.py

```diff
@@ -0,0 +1,12 @@
+from _exampleLib import *
```

Comment:
Changed. Although in the future I think we will need to make an exception for this particular case (wrapped C++ modules that need extending in Python).